app/services/bucket_service.rb

```diff
@@ -8,9 +9,11 @@ def self.send_bucket_created_emails(bucket: )
   end
 
   def self.send_bucket_live_emails(bucket: )
-    memberships = bucket.group.memberships.active.reject { |membership| membership.member == bucket.user }
-    memberships.each do |membership|
-      member = membership.member
+    group = bucket.group
+    members = bucket.participants(exclude_author: true, subscribed: true)
+    members.each do |member|
```

Comment:
you might be quite right. 

mmm

```ruby
scope :published_for_group, ->(group) { joins(:membership).where(memberships: {archived_at: nil, group_id: group.id}) }
```

I think that would work ... it's a fancy join I learnt from Chris T.
Anyway not sure if it's a worthwhile complexity here...